x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_charts/risks_table.tsx

```diff
@@ -90,7 +90,9 @@ export const RisksTable = ({
             compact
               ? css`
                   thead {
-                    display: none;
+                    .euiTableCellContent {
+                      padding: 0;
+                    }
```

Comment:
previous solution that removed the `thead` completely had broken the width we set in the `columns` property